lookout/style/format/analyzer.py

```diff
@@ -38,6 +39,9 @@ class FormatAnalyzer(Analyzer):
         "report_code_lines": True,
         "report_triggered_rules": True,
         "report_parse_failures": True,
+        "report_github_suggestion": True,
+        "report_error_description": True,
+        "uast_break_check": True,
```

Comment:
We need a Jinja2 template for our comments. There should be the default one and the ability to change it in config.